commands/init.go

```diff
@@ -1,27 +0,0 @@
-package commands
```

Comment:
For commit "refactor commands to configuration", split it in multiple commit, like:

one commit te rework on base of configuration and one commit by configuration type (database, ...)

Missing unit for each configurations.

Reply:
Separate into different commits: ok
Unit tests for each configuration: after a refactoring there is a new version in which each of the configurations has no logic, they are a simple list of keys.